ably/state.go

```diff
@@ -29,7 +29,7 @@ func (st StateType) String() string {
 	case StateChan:
 		return "channel"
 	default:
-		return ""
+		return "invalid"
```

Comment:
@kouno: Logger is implemented and handled in ClientOptions, but I didn't deployed it yet - that is I'm going to add logging after the lib is finished, to not keep reworking the logging.

While I agree exceptional, but non-vital, situations should be logged, we can't log from the inside of the model. At least I think so.